src/multiversioning.zig

```diff
@@ -390,6 +400,15 @@ pub const MultiversionHeader = extern struct {
         if (!self.current_flags.visit) return error.InvalidCurrentFlags;
         if (self.current_release == 0) return error.InvalidCurrentRelease;
 
+        // current_git_commit and current_release_client_min were added after 0.15.4.
+        if (self.current_release > (try Release.parse("0.15.4")).value) {
```

Comment:
If we keep adding things like this, this'll be a whole bunch of if statements. Maybe that's ok?